theme.js

```diff
@@ -42,6 +78,10 @@ function loadTheme() {
         for (const [key, value] of Object.entries(colors)) {
           document.body.style.setProperty("--" + key, value);
         }
+        const strippedTheme = theme.split('_').join('').toLowerCase();
```

Comment:
> What I don’t understand is the second if statement when checking for the magic word. It only triggers when the name starts with letitsnow exactly, so why is it needed?

Are you looking at the most recent version? I had an if statement checking for the words at the beginning of the name, but I removed that to make it easier to trigger the easter egg.

This new check just makes it so all of these theme names should trigger it:
* "Let It Snow Theme"
* "Word let it snow"
* "Theme letitsnow 2"
* "leTitsnOw example"